### PR TITLE
Small fixes for ASL Reference

### PR DESCRIPTION
**Pull request by yuxiliu-arm**
These are flagged by `pandoc` and may also make sense for the upstream LaTeX tool.

### CODE REVIEW

**Review by Roman-Manevich — APPROVED** [review]
Looks good to me.

File: asllib/doc/LexicalStructure.tex
Comment (yuxiliu-arm) [inline asllib/doc/LexicalStructure.tex:1]: Actually, this seems like a MathJax error, it reports `\textbackslash` is only available in math mode, while it should be available in text mode. Will revert this change.

**Review by Roman-Manevich — APPROVED** [review]
Looks good to me!
